src/autorest-core/lib/autorest-core.ts

```diff
@@ -167,9 +167,14 @@ export class AutoRest extends EventEmitter {
         }
 
         if (view.InputFileUris.length === 0) {
-          throw new Exception("No input files provided.\n\nUse --help to get help information.", 0);
+          if (view.GetEntry("allow-no-input")) {
```

Comment:
Shouldn't we display help if invalid inputs are provided. Not a blocker, just curious.